Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Generic/EquipmentRotation/Subsequences/Validation/Two_uRot.mo

```diff
@@ -0,0 +1,55 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Generic.EquipmentRotation.Subsequences.Validation;
+model Two_uRot
+  "Validation sequence for device role rotation for two devices or groups of devices"
+
+  Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Generic.EquipmentRotation.Subsequences.Two rotTwo
+    "Rotates roles between two devices or groups of devices"
+    annotation (Placement(transformation(extent={{20,20},{40,40}})));
+
+  Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Generic.EquipmentRotation.Subsequences.Two rotTwo1
+    "Rotates roles between two devices or groups of devices"
+    annotation (Placement(transformation(extent={{20,-40},{40,-20}})));
+
+protected
+  Buildings.Controls.OBC.CDL.Logical.Sources.SampleTrigger rotation(period(
+    final displayUnit="h") = 86400) "Rotation trigger"
```

Comment:
It would be better to set the `startTime` other than the default zero, say set it `startTime=7200` so that it will show the value of `initialRoles`.